tests/migrations/test_operations.py

```diff
@@ -51,7 +51,7 @@ def make_test_state(self, app_label, operation, **kwargs):
         return project_state, new_state
 
     def set_up_test_model(
-            self, app_label, second_model=False, third_model=False,
+            self, app_label, second_model=False, third_model=False, index=False,
```

Comment:
Also, single column indexes are being tested in `test_add_index` [here](https://github.com/django/django/pull/6726/files#diff-5210414c7a29d5d1f61588388b517599R1343).